atd-toolbox/read_replica_proxy/entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+envsubst < /usr/local/etc/haproxy/haproxy.cfg.template > /usr/local/etc/haproxy/haproxy.cfg
```

Comment:
This takes the environment variables of the docker container once it starts up and substitutes them into the `haproxy.cfg.template` file. The result is what's used when we fire up `haproxy` on the next line.